src/tests/running.md

```diff
@@ -93,3 +93,24 @@ This is much faster, but doesn't always work. For example, some tests
 include directives that specify specific compiler flags, or which rely
 on other crates, and they may not run the same without those options.
 
+### Run only the tidy script
```

Comment:
Perhaps move these to the end of the "Running a subset of the test suites" section above (in the same file).

Reply:
This is done.